operator/config/watcher/watcher.yaml

```diff
@@ -0,0 +1,23 @@
+# labels and selectors will be added using kustomize
+apiVersion: operator.kyma-project.io/v1alpha1
+kind: Watcher
+metadata:
+  name: watcher
+spec:
+  componentName: lifecycle-manager
+  contractVersion: v1
+  gvrsToWatch: [] # needs to be removed after removing it in the runtime-WatcherCRD
+  serviceInfo:
+    serviceName: lifecycle-manager-event-service
+    servicePort: 8082
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: event-service
+spec:
+  ports:
```

Comment:
This service is missing a selector

Reply:
Thats why I added the comment, that the selector and labels will be added using kustomize. If you call `make deploy` you will see that the selectors will be added

Reply:
yes but as above, the services should be appliable without kustomize in its default form. This is why if you run without the transformer it will fail

Reply:
I will add them to make it work, if kustomize is not being used